qubes/tools/qvm_block.py

```diff
@@ -216,13 +233,41 @@ def get_parser():
     attach_parser.add_argument(metavar='POOL_NAME:VOLUME_ID', dest='volume',
                                action=qubes.tools.VolumeAction)
     attach_parser.set_defaults(func=attach_volumes)
+
+
+def init_dettach_parser(sub_parsers):
     detach_parser = sub_parsers.add_parser(
         "detach", help="Detach volume from domain", aliases=('d', 'dt'))
     detach_parser.add_argument('VMNAME', action=qubes.tools.RunningVmNameAction)
     detach_parser.add_argument(metavar='POOL_NAME:VOLUME_ID', dest='volume',
                                action=qubes.tools.VolumeAction)
     detach_parser.set_defaults(func=detach_volumes)
 
+def init_extend_parser(sub_parsers):
+    extend_parser = sub_parsers.add_parser(
+        "extend", help="extend volume from domain", aliases=('d', 'dt'))
+    extend_parser.add_argument(metavar='POOL_NAME:VOLUME_ID', dest='volume',
+                               action=qubes.tools.VolumeAction)
+    extend_parser.add_argument(
+        'size', help='size in bytes of the new ThinPoolLogicalVolume')
```

Comment:
I think `ThinPoolLogicalVolume` here is too technical (and inaccurate for non-LVM) here.